### DISCUSSION TITLE
Mountain duck: unable to open cache directory in finder

### DISCUSSION THREAD

**Discussion by minfrin** [iterate-ch/cyberduck#14320] [Q&A]
**Describe the bug**
In the preferences, the mount location and cache directories are specified, but there is no "open in finder" option.

This makes it difficult to delete the cache directory, as it has to be navigated to manually one level at a time.

**To Reproduce**
1. Open preferences, sync
2. Click on "Cache"

**Expected behavior**
An option "open in finder" so the cache directory can be opened in finder.

**Screenshots**
<img width="712" alt="image" src="https://user-images.githubusercontent.com/1331788/214113600-575bf7e9-e2b1-4722-b4bb-e31484efd5de.png">
<img width="584" alt="image" src="https://user-images.githubusercontent.com/1331788/214113798-20662073-e8b2-4e6d-94d5-bde2477984e5.png">

**Desktop (please complete the following information):**
 - OS: MacOS

**Log Files**

**Additional context**

**Comment by dkocher** [comment]
Choosing _Delete Bookmark_ will move the cache directory to the trash.

**Comment by minfrin** [comment]
Definitely don't want to delete the bookmark - just want to see where the cache is, so that I can see how big it is, and to blow it away.

**Comment by tatze96** [comment]
You can change or see the cache location by clicking on _Choose..._ after clicking on _Cache_ within the Mountain Duck preferences _Sync_ tab.

The cache directory is by default located in `~/Library/Group Containers/G69SCX94XU.duck/Library/Application Support/duck/Cache`.

You can reach the directory by choosing `Go → Go to folder` within the Finder menu, copying the path into the appearing window, and clicking on the _Open_ button afterward.

I don't recommend deleting the cache folder, as this can lead to file loss if there are synchronization errors, conflicts, or files that aren't fully synchronized yet.

Instead, you can manually free up some disk space by right-clicking on a file or folder within the mounted drive and choosing `Mountain Duck → Delete on Local Disk` to clear out the file(s) from the cache. You can do the same while right-clicking on white space within the mount to clear out all files from the cache. Note that this will also clear out the files kept offline purposely.

Using the _Delete on Local Disk_ option won't prevent Mountain Duck from downloading the files to the cache again.
